tests/eosio.rentbw_tests.cpp

```diff
@@ -350,37 +455,52 @@ struct rentbw_tester : eosio_system_tester
          ilog("after_reserve.net:                        ${x}", ("x", after_reserve.net));
          ilog("before_reserve.cpu:                       ${x}", ("x", before_reserve.cpu));
          ilog("after_reserve.cpu:                        ${x}", ("x", after_reserve.cpu));
-
-         csv->newRow() << last_block_time()
-                       << before_state.net.assumed_stake_weight
-                       << before_state.net.weight_ratio / double(rentbw_frac)
-                       << before_state.net.weight
-                       << before_receiver.net
-                       << after_receiver.net
-                       << after_receiver.net - before_receiver.net
-                       << before_payer.liquid - after_payer.liquid
-                       << before_reserve.net
-                       << after_reserve.net
-                       << before_reserve.cpu
-                       << after_reserve.cpu
-
-                       << get_state().cpu.weight
-                       << get_state().cpu.weight_ratio
-                       << get_state().cpu.assumed_stake_weight
-                       << get_state().cpu.initial_weight_ratio
-                       << get_state().cpu.target_weight_ratio
-                       << get_state().cpu.initial_timestamp.sec_since_epoch()
-                       << get_state().cpu.target_timestamp.sec_since_epoch()
-                       << get_state().cpu.exponent
-                       << get_state().cpu.decay_secs
-                       << get_state().cpu.min_price.to_string()
-                       << get_state().cpu.max_price.to_string()
-                       << get_state().cpu.utilization
-                       << get_state().cpu.adjusted_utilization
-                       << get_state().cpu.utilization_timestamp.sec_since_epoch()
-                       << get_state().cpu.fee
-
-                       << (max > 0 ? "rentbwexec" : "rentbw");
+   
+         csv->newRow() << last_block_time()           
+                      << before_state.net.assumed_stake_weight
+                      << before_state.net.weight_ratio / double(rentbw_frac)
+                      << before_state.net.weight
+                      << float(before_reserve.net / 10000.0)
```

Comment:
console output (line 454) is different. Should it be changed as well?

Reply:
updated some of them.

Reply:
fix these lines as well if you end up using `to_human_readable()` from my other comment